DEVELOPMENT.md

```diff
@@ -103,6 +103,8 @@ Once you've done this, merge or rebase your development branch with config chang
 
 * Finally, before committing your final changes and opening your pull request, be sure and run `make test` locally. This will regenerate the JSON schemas for the skaffold config with your new changes. Commit the resulting changes autogenerated by the scripts.
 
+For more details behind the logic of config changes see [the Skaffold config management doc](https://docs.google.com/document/d/e/2PACX-1vRBN1vHjSGbs3ZmdvyzzhBn48NjeuP4OiTTG5mec_YrRzNDL63pFa1Xy1rgCUNiQXTjAE1QyvCr5yxX/pub).
```

Comment:
Is the document supposed to be public? I can't access it

Reply:
It is supposed to be public, huh, I'll update.